ql/src/test/results/clientpositive/llap/ppd_field_garbage.q.out

```diff
@@ -17,7 +17,7 @@ POSTHOOK: Input: default@test_issue
 POSTHOOK: Output: database:default
 POSTHOOK: Output: default@v_test_issue
 POSTHOOK: Lineage: v_test_issue.age EXPRESSION [(test_issue)test_issue.FieldSchema(name:test_c, type:struct<user_c:struct<age:int>>, comment:null), ]
-POSTHOOK: Lineage: v_test_issue.fileid EXPRESSION [(test_issue)test_issue.FieldSchema(name:fileid, type:int, comment:null), ]
```

Comment:
I am not very familiar with lineage to comment if this change from `EXPRESSION` to `SIMPLE` is an improvement or a regression. I checked a bit `org.apache.hadoop.hive.ql.hooks.LineageInfo.DependencyType` but not really sure what should be the type here. Do you happen to know something more about it?